src/pages/About/AboutPage.jsx

```diff
@@ -0,0 +1,282 @@
+                  Did you find a box out in the world? Did you receive a box from someone?
+                </Text>
+                <Text className={styles['about-text-center']}>
+                  <strong>First of all, welcome to the Find Your Anchor (FYA) community!</strong>
+                  <br />
+                  <br />
+                  This box was meant to find its way to you and we hope you can feel all the love
+                  packed in there.
+                  <br />
+                  <br />
+                  There are people all over the world who care.
+                </Text>
+                <Text>
+                  <strong>
+                    If you&apos;d like, you can&nbsp;
+                    <a href="/pickup-box-form" style={{ color: '#2B6CB0' }}>
+                      claim
+                    </a>
+                    &nbsp;this box on the map! All you need is a picture of the box number (located
+                    in the upper left of the box) and a zip code.
+                  </strong>
+                </Text>
+                <div className={styles['privacy-note-section']}>
+                  <Text>
+                    Once a box is launched, only the zip code is displayed (unless you share a
+                    photo!).
+                  </Text>
+                  <div className={styles['privacy-note']}>
+                    <img src={PrivacyNoteIcon} alt="" />
+                    <Text className={styles['privacy-note-text']}>
+                      <em>
+                        Please know we will never display, share, or sell your personal information,
+                        so please do not include anything identifying yourself. Email will only be
+                        used for submission status updates.
+                      </em>
+                    </Text>
+                  </div>
+                  <Text>We&apos;re strangers who care - let&apos;s keep it that way! ;)</Text>
+                </div>
+                <Text>
+                  <strong>
+                    For any questions, please email us at&nbsp;
+                    <a href="mailto:hello@findyouranchor.us" style={{ color: '#2B6CB0' }}>
+                      hello@findyouranchor.us.
+                    </a>
+                  </strong>
+                </Text>
+              </TabPanel>
+            </TabPanels>
+          </Tabs>
+        </div>
+      </div>
+
+      <div className={styles.footer}>
+        <Button
+          className={styles['login-button']}
+          onClick={navigateToLogin}
+          color="white"
+          bg="#345E80"
+          size="lg"
+        >
+          Admin Login
+        </Button>
+        <Button
+          className={styles['login-button-mobile']}
+          onClick={navigateToLogin}
+          color="white"
+          bg="#345E80"
+          size="md"
+        >
+          Admin Login
+        </Button>
+        <div className={styles['social-icon-section']}>
```

Comment:
On one of the About pages on the Figma (on the topmost General User Side section), they also include the website and email in this section. Could you confirm with designers if these are needed and if so, could you add them here?

Reply:
Confirmed with designers during the teamwide meeting - the website and email should be added so please do so :)

Reply:
Fixed this and also went ahead and moved footer into its own component, since designers have added it to some of the other general user pages on the figma.

Reply:
cool tyty